app/gateway/changelog.md

```diff
@@ -9,6 +9,39 @@ Changelog for supported Kong Gateway versions.
 
 For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).
 
+## 3.4.2.0 
+**Release date** 2023/11/08
+
+### Features
+#### Enterprise
+
+* License management:
+  * Implemented a new grace period that lasts 30 days from the Kong Enterprise license expiration date. 
+    During the grace period all open source functionality will be available, and
+    Enterprise functionality will be set to read-only mode.
+  * Added support for counters such as routes, plugins, licenses, and deployment information to the license report.
+  * Added a checksum to the output of the license endpoint.
+### Fixes
+#### Core
+* DNS
+  * Fixed an issue with the DNS client was not adhering to configured timeouts in a predictable manner. [#11386](https://github.com/Kong/kong/pull/11386)
+  * Fixed an issue that caused the DNS client to resolve incorrectly during transient network and DNS server failures. [#11386](https://github.com/Kong/kong/pull/11386)
```

Comment:
These are the same issue with two different entries (see the link to the PR). You should combine them into one.